WooCommerce/Classes/ViewRelated/AppCoordinator.swift

```diff
@@ -166,18 +172,43 @@ private extension AppCoordinator {
         setWindowRootViewControllerAndAnimateIfNeeded(tabBarController)
     }
 
-    /// If the app is authenticated but there is no default store ID on launch: Let's display the Store Picker.
+    /// If the app is authenticated but there is no default store ID on launch,
+    /// check for errors and display store picker if none exists.
     ///
-    func displayStorePicker() {
+    func handleLoggedInStateWithoutDefaultStore() {
```

Comment:
nit: maybe we can keep the `display*` to indicate that this is to display some UI?

```suggestion
    func displayLoggedInStateWithoutDefaultStore() {
```

Reply:
Updated in 7846969070aea77aa19b3fda0c0c05e3441c28f3.